.github/workflows/publish.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           repository: getsentry/${{ fromJSON(steps.inputs.outputs.result).repo }}
           token: ${{ secrets.GH_SENTRY_BOT_PAT }}
           fetch-depth: 0
+          ref: ${{ fromJSON(steps.inputs.outputs.result).version }}
```

Comment:
I'm not sure this is right -- I think the expectation is that you'd make the changes to the craft yaml on the main branch (since that would require you to go through a PR with the proper branch protection)

Reply:
But then, that means we need to modify the craft file in the main and leave it in a broken state, because the rest of the project wouldn't reflect the necessary changes for the new craft to work.
We want to release a beta version of the SDK to test it properly, but if we change the craft file in the main, we'll need to revert it in case we need to release a hotfix.
This seems like a very strange workflow.

Reply:
yeah I agree it's weird, and I'm not trying to take the side of craft here -- but I could also imagine a rogue actor who maliciously modifies `craft.yml` in a release branch

Reply:
Well, we solve this by changing the repo main branch. 
I will close this them. 

Thanks